src/main/java/io/strimzi/kafka/quotas/Volume.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.quotas;
+
+import java.util.Objects;
+
+/**
+ * Represents a single volume on a specific Kafka broker.
```

Comment:
I really represents the _usage_ of a volume on a broker. So maybe `VolumeUsage`?

Reply:
`VolumeUsage` is certainly better than the other names that we messed around with. I think I discounted `usage` as too ambiguous but thinking about it now there isn't really anything else it could realistically be referring to. 

Will update.

Reply:
Resolved with https://github.com/strimzi/kafka-quotas-plugin/pull/33/commits/824fd3e3a119b73d943f1375928da39564f88c86